src/evaluate.ts

```diff
@@ -191,18 +197,21 @@ async function loadFile(document, callback: () => { }, pprintOptions: PrettyPrin
         })
         await res.value.then((value) => {
             if (value) {
+                setEvalStatus(EvaluationStatus.success);
                 chan.appendLine("=> " + value);
             } else {
+                setEvalStatus(EvaluationStatus.none);
                 chan.appendLine("No results from file evaluation.");
             }
         }).catch((e) => { 
             chan.appendLine(`Evaluation of file ${fileName} failed: ${e}`);
+            setEvalStatus(EvaluationStatus.error);
         });
     }
     if (callback) {
```

Comment:
This was recently added, I'm not familiar with the purpose.  Would it make sense for this to also be represented in the status bar indicator?

Reply:
I think this might be from @cfehse 's fix for the load file callback. And yes, would probably be material for a status indicator.

Reply:
Yes it is.

Reply:
The purpose was solely to show that an error occurred instead of ignoring the error condition completely.